gms/impl/src/main/java/com/linkedin/metadata/resources/dataset/GlossaryTermsResource.java

```diff
@@ -0,0 +1,35 @@
+package com.linkedin.metadata.resources.dataset;
+
+import com.linkedin.common.GlossaryTerms;
+import com.linkedin.parseq.Task;
+import com.linkedin.restli.server.CreateResponse;
+import com.linkedin.restli.server.annotations.RestLiCollection;
+import com.linkedin.restli.server.annotations.RestMethod;
+
+import javax.annotation.Nonnull;
+
+
+/**
+ * Rest.li entry point: /datasets/{datasetKey}/glossaryTerms
+ */
```

Comment:
These aspect-specific resources were created for the case where someone would want to specifically request a single aspect of an entity. They are not commonly used. Unless you have a specific need to just fetch the glossaryTerms of a dataset and no other information, I would recommend not including this file

Reply:
Initially we thought of having this API to get term linked to dataset, but I think in the UI snapshot data will be available and it will consist business term data as well. We can remove this